### PR TITLE
Calendars menus

### PR DESCRIPTION
**Pull request by rsvato**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -49,23 +49,40 @@
 	# Set the URL pointing to the bundle Zip to download. If the URL points to a
 	# DXP bundle (e.g., https://api.liferay.com/...), set the
 	# "liferay.workspace.bundle.token.download" property to true. The default
-	# value is the URL for Liferay Portal CE 7.0 GA7.
+	# value is the URL for Liferay Portal CE 7.0 GA7, Liferay Portal CE 7.1 GA4,
+	# Liferay Portal CE 7.2 GA2, or Liferay Portal CE 7.3 GA1, depending on the
+	# portal version the workspace is targeting.
 	#
-	liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.1.0-ga4/liferay-ce-portal-tomcat-7.1.3-ga4-20190508171117552.7z
+	liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.2.0-ga1/liferay-ce-portal-tomcat-7.2.0-ga1-20190531153709761.tar.gz
```
Comment (adiukov) [inline gradle.properties]: liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.2.1-ga2/liferay-ce-portal-tomcat-7.2.1-ga2-20191111141448326.tar.gz

File: gradle.properties [hunk 2]
```diff
@@ -106,12 +123,25 @@
...
+	# Set the Liferay Portal or DXP version to develop and test against. By
+	# setting this property, it enables the target platform features such as
+	# dependency management and OSGi resolve tasks. Use the version that
+	# matches the Liferay Portal or DXP bundle version in this workspace.
+	#
+	# For a list of all available target platform versions, see
+	# https://bit.ly/2IkAwwW for Liferay Portal and https://bit.ly/2GIyfZF for
+	# Liferay DXP.
+	#
+	liferay.workspace.target.platform.version=7.2.0
```
Comment (adiukov) [inline gradle.properties]: liferay.workspace.target.platform.version=7.2.1

File: gradle.properties (hunk 1 above)
Comment (rsvato) [reply gradle.properties]: Fixed

File: gradle.properties (hunk 2 above)
Comment (rsvato) [reply gradle.properties]: Fixed

**Pull request merged**